livechat/agent/web/base.py

```diff
@@ -41,6 +42,7 @@ def get_client(
                                verify the identity of requested hosts. Either `True` (default CA bundle),
                                a path to an SSL certificate file, an `ssl.SSLContext`, or `False`
                                (which will disable verification). Defaults to `True`.
+                disable_logging (bool): A boolean indicating if logging should be disabled.
```

Comment:
```suggestion
                disable_logging (bool): indicates if logging should be disabled.
```
"A boolean" is redundant since it's mentioned in the parenthesis. Applies to all occurrences.